elements/pfe-sass/mixins/_copy-mixins.scss

```diff
@@ -13,3 +13,21 @@
     text-decoration: underline;
     text-decoration-skip: ink;
 }
+@mixin pfe-heading-xl($size: "FontSize--4xl") {
```

Comment:
I think these need a hook for local too so that we can use these in components.

Reply:
I updated this whole mixin & included a local hook